ui/src/main/java/de/ipb_halle/lbac/items/bean/aliquot/createaliquot/CreateAliquotStep3Controller.java

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
How does that class differs with the step X Controller from solutions? Can both classes be united to one?